deploy/ansible/roles/iroha-cluster-config-gen/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Why iroha/deploy/ansible/roles/iroha-cluster-config-gen/tasks/main.yml does not have ubuntu.yml file?

Reply:
@neewy will be fixed in next commit